src/shared/unit-file.c

```diff
@@ -329,8 +329,9 @@ int unit_file_build_name_map(
                                         continue;
                                 }
 
-                                if (!path_is_absolute(target)) {
-                                        target_abs = path_join(*dir, target);
+                                const bool is_abs = path_is_absolute(target);
```

Comment:
what does the "const" du here? it's very confusing suddently using that for stuff that isn't a ptr...

Reply:
I'm just saying that this a variable that will not be modified later on. A hint for the reader.

Reply:
I can remove it if you think it's absolutely wrong...